### PR TITLE
.NET SDK for Oracle.Database

### PR DESCRIPTION
**Pull request by v-jamcheung**
.NET SDK for Oracle.Database

Library Namespace: Azure.ResourceManager.OracleDatabase (https://github.com/Azure/azure-sdk/issues/7507)
Merged PR for Oracle.Database Spec: https://github.com/Azure/azure-rest-api-specs/pull/29305

### CODE REVIEW

**Comment by v-jamcheung** [comment]
@ArthurMa1978 @KrzysztofCwalina @tg-msft @archerzz @ArcturusZhang @pallavit @ronniegeraghty

Hi team,

Please could you review this PR and leave comments if there are any issues.

Thanks.

**Review by jsquire — DISMISSED** [review]
@v-jamcheung :

Your account lacks the permissions required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

**Comment by jsquire** [comment]
@v-jamcheung:  Please help me understand why we need two pull requests active for this set of changes.   Please close either this PR or #44106.

**Comment by vsramki** [comment]
> @v-jamcheung: Please help me understand why we need two pull requests active for this set of changes. Please close either this PR or #44106.

@jsquire The old one has now been closed

**Comment by jsquire** [comment]
> > Member
> 
> @jsquire Jamie and I belong to the Oracle Team. Jamie's request to be a contributor is pending PM approval while I have those permissions already. This also reflect in the CODEOWNERS file where only my account has a mention. Could this PR be considered on that basis while we get Jamie the necessary permissions. Thank you.

Given that we've got a CODEOWNER that won't break automation and an active request for permissions, I'm good with this.   I'll remove my block on the PR so that you can work through the actual changes with Arthur and team.

**Comment by v-jamcheung** [comment]
> @v-jamcheung :
> 
> Your account lacks the permissions required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

My account now has the permissions required of an internal contributor.
Verification:
Screenshot 2024-06-14 at 09 50 49

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -972,10 +972,10 @@
 # ServiceOwners:                                                   @jamesfan1 @bangbingsyb @spencerofwiti
 
 # PRLabel: %Oracle
-/sdk/oracle/Azure.ResourceManager.*/                               @v-rvilathurs
+/sdk/oracle/Azure.ResourceManager.*/                               @v-jamcheung @v-rvilathurs @eelhomsi1
```
Comment (jsquire) [inline .github/CODEOWNERS:975]: `@eelhomsi1` is not a valid GitHub name.

**Review by jsquire — DISMISSED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (v-jamcheung) [reply .github/CODEOWNERS]: Updated

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -972,10 +972,10 @@
 # ServiceOwners:                                                   @jamesfan1 @bangbingsyb @spencerofwiti
 
 # PRLabel: %Oracle
-/sdk/oracle/Azure.ResourceManager.*/                               @v-jamcheung @v-rvilathurs
+/sdk/oracle/Azure.ResourceManager.*/                               @v-jamcheung @v-rvilathurs @v-eelhomsi
```
Comment (jsquire) [inline .github/CODEOWNERS:975]: v-eelhomsi requires public Microsoft and Azure memberships.   But, hey - at least they exist now!

Comment (v-eelhomsi) [reply .github/CODEOWNERS:975]: Not only do I now exist - I'm officially public!

Comment (jsquire) [reply .github/CODEOWNERS:975]: Confirmed!  You're real and official.

File: eng/Packages.Data.props [hunk 3]
```diff
@@ -121,7 +121,7 @@
     <PackageReference Update="Azure.MixedReality.Authentication" version= "1.2.0" />
     <PackageReference Update="Azure.Monitor.OpenTelemetry.Exporter" Version="1.3.0" />
     <PackageReference Update="Azure.Monitor.Query" Version="1.1.0" />
-    <PackageReference Update="Azure.Identity" Version="1.11.1" />
+    <PackageReference Update="Azure.Identity" Version="1.11.4" />
```
Comment (jsquire) [inline eng/Packages.Data.props:124]: This shouldn't be here.   Rebasing on main will bring this change in.

**Comment by jsquire** [comment]
@ArthurMa1978:  Central + Repo content is in a good state.   All you for the management library parts.

File: .github/CODEOWNERS (hunk 1 above)
Comment (ArthurMa1978) [reply .github/CODEOWNERS]: @v-jamcheung please join the https://github.com/orgs/Azure/teams/azure-sdk-partners

File: sdk/oracle/Azure.ResourceManager.OracleDatabase/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure OracleDatabase management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/oracle/Azure.ResourceManager.OracleDatabase/README.md:3]: Describe your service here

File: sdk/oracle/Azure.ResourceManager.OracleDatabase/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/oracle/Azure.ResourceManager.OracleDatabase/CHANGELOG.md:3]: Specify the expected release date by replacing the `Unreleased`, the format is `yyyy-mm-dd`.

**Review by ArthurMa1978 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (ArthurMa1978) [reply .github/CODEOWNERS]: done

File: sdk/oracle/Azure.ResourceManager.OracleDatabase/Azure.ResourceManager.OracleDatabase.sln [hunk 6]
```diff
@@ -0,0 +1,65 @@
+﻿
```
Comment (weshaggard) [inline sdk/oracle/Azure.ResourceManager.OracleDatabase/Azure.ResourceManager.OracleDatabase.sln:1]: @ArthurMa1978 all the other languages have this under `sdk/oracledatabase` is there any reason .NET cannot follow the same?

**Pull request merged**